src/main/Java/AnagramKata.java

```diff
@@ -0,0 +1,50 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class AnagramKata {
+    public static Scanner s;
+
+    public AnagramKata(String path) throws FileNotFoundException {
+        File wordList = new File(path);
+        s = new Scanner(wordList);
+    }
+
+    public Integer returnsNumberOfAnagrams(String word) throws FileNotFoundException {
+        Integer totalNumber = 0;
+        while (s.hasNext()) {
+            totalNumber += checksWordsForAnagram(s.next(), word);
+        }
+        s.close();
+        return totalNumber;
+    }
+
+
+
+    public Integer checksWordsForAnagram(String wordToCheck, String secondWordToCheck) throws FileNotFoundException {
+        if (splitIntoSet(wordToCheck.toLowerCase().replaceAll("[^a-z]", "")).equals(splitIntoSet(secondWordToCheck.toLowerCase().replaceAll("[^a-z]", "")))) {
+            return 1;
+        }
+        return 0;
+    }
+
+    public Map splitIntoSet(String wordToGoIntoSet) {
```

Comment:
The name talks about sets but it's been changed to deal with maps so it's a bit misleading, needs a better name.